openshift/auth.config.yaml

```diff
@@ -29,7 +29,7 @@ objects:
   type: Opaque
   data:
     service-account-secrets: >-
-      ewogICJhY2NvdW50cyI6IFsKICAgICAgewogICAgICAgICAgIm5hbWUiOiJmYWJyaWM4LXdpdCIsCiAgICAgICAgICAiaWQiOiI1ZGVjNWZkYi0wOWUzLTQ0NTMtYjczZi01YzgyODgzMmIyOGUiLAogICAgICAgICAgInNlY3JldHMiOlsid2l0c2VjcmV0Il0KICAgICAgfSwKICAgICAgewogICAgICAgICAgIm5hbWUiOiJmYWJyaWM4LXRlbmFudCIsCiAgICAgICAgICAiaWQiOiJjMjExZjFiZC0xN2E3LTRmOGMtOWY4MC0wOTE3ZDE2Nzg4OWQiLAogICAgICAgICAgInNlY3JldHMiOlsidGVuYW50c2VjcmV0T2xkIiwgInRlbmFudHNlY3JldE5ldyJdCiAgICAgIH0KICBdCn0=  
+      ew0KICAiYWNjb3VudHMiOiBbDQogICAgICB7DQogICAgICAgICAgIm5hbWUiOiJmYWJyaWM4LXdpdCIsDQogICAgICAgICAgImlkIjoiNWRlYzVmZGItMDllMy00NDUzLWI3M2YtNWM4Mjg4MzJiMjhlIiwNCiAgICAgICAgICAic2VjcmV0cyI6WyJ3aXRzZWNyZXQiXQ0KICAgICAgfSwNCiAgICAgIHsNCiAgICAgICAgICAibmFtZSI6ImZhYnJpYzgtdGVuYW50IiwNCiAgICAgICAgICAiaWQiOiJjMjExZjFiZC0xN2E3LTRmOGMtOWY4MC0wOTE3ZDE2Nzg4OWQiLA0KICAgICAgICAgICJzZWNyZXRzIjpbInRlbmFudHNlY3JldE9sZCIsICJ0ZW5hbnRzZWNyZXROZXciXQ0KICAgICAgfSwNCiAgICAgIHsNCiAgICAgICAgICAgICJuYW1lIjoib25saW5lLXJlZ2lzdHJhdGlvbiIsDQogICAgICAgICAgICAiaWQiOiJmODY3ZWM3Mi0zMTcxLTRiOGYtOGVlYy05MGEzMmVhYjZlMGIiLA0KICAgICAgICAgICAgInNlY3JldHMiOlsicmVnaXN0cmF0aW9uc2VjcmV0Il0NCiAgICAgIH0NCiAgXQ0KfQ==  
```

Comment:
We don't really need to update this every time we add a new service account. This particular secret is not used anywhere. It's just an example.

Reply:
Thanks - I was wondering we might need these for end-to-end testing?

Reply:
openshift/auth.config.yaml is not used anywhere. It's just an example. Real secrets are set in our clusters.
So, for Auth tests (unit and integration) /configuration/config-files/service-account-secrets.conf is used. For end-to-end tests the secrets set in our preview will be used.

Reply:
Got it, 1947401788237363984b0b803839c29d74696441